### DISCUSSION TITLE
Add an argon-themed judgement counter

### DISCUSSION THREAD

**Discussion by felipemarins** [ppy/osu#30407] [Ideas]
I don't know if this is the most appropriate way to request new skin elements. But I think that the current triangles judgement counter is one of the most out of place components when combined with the argon skin, so I would like if there was a dedicated argon component for that.

If this is the right figma document to look at, it looks like it would be pretty similar to the key counter: https://www.figma.com/design/Tu1C6CR2XOdhm7kxq1JQNj/Gameplay-Update?node-id=508-78 (I found it on #27498)

A little obvious adjustment to it would be to colour each of the horizontal lines above the judgements according to the colours used for them on argon. With that, I guess the judgement names could be removed (or maybe add the option to do that) as they would be a bit redundant.

**Comment by mcendu** [comment]
> With that, I guess the judgement names could be removed (or maybe add the option to do that) as they would be a bit redundant.

Colorblind players should be taken into account. Keeping the judgement names seems to be the easiest solution.

**Reply by felipemarins** [reply]
Sorry, I didn't consider that when I was writing this discussion. Then I think that the judgement labels should not be removed entirely, but I still would like if there was an option to hide them, if possible.
